scripts/hostcfgd

```diff
@@ -1071,16 +1082,27 @@ class PamLimitsCfg(object):
         self.config_db = config_db
         self.hwsku = ""
         self.type = ""
+        self.max_sessions = None
 
     # Load config from ConfigDb and render config file/
     def update_config_file(self):
         device_metadata = self.config_db.get_table('DEVICE_METADATA')
-        if "localhost" not in device_metadata:
+        ssh_server_policies = self.config_db.get_table('SSH_SERVER')
+
+        if ("localhost" not in device_metadata) and ("POLICIES" not in ssh_server_policies):
+            syslog.syslog(syslog.LOG_ERR,
+                "didnt modified config, exiting. {}", ssh_server_policies)
```

Comment:
Please add more readable error. I believe error in this situation should be "Invalid configuration. Missing POLICIES in SSH_SERVER config"